tests/system/cases/windows_service/main.tf

```diff
@@ -7,7 +7,8 @@ provider "layer0" {
 resource "layer0_environment" "ws" {
   name = "ws"
   os   = "windows"
-  size = "m3.large"
+  instance_type = "m3.large"
```

Comment:
this needs a scale of at least 1 to pass, right?

Reply:
Yes, you're correct. ✅